src/main/java/com/google/devtools/build/lib/rules/cpp/CppOptions.java

```diff
@@ -1004,6 +1004,25 @@ public Label getPropellerOptimizeLabel() {
       help = "If enabled, write CppCompileAction exposed action.args to parameters file.")
   public boolean useArgsParamsFile;
 
+  @Option(
+      name = "experimental_unsupported_and_brittle_include_scanning",
+      defaultValue = "false",
+      documentationCategory = OptionDocumentationCategory.BUILD_TIME_OPTIMIZATION,
+      effectTags = {
+        OptionEffectTag.LOADING_AND_ANALYSIS,
+        OptionEffectTag.EXECUTION,
+        OptionEffectTag.CHANGES_INPUTS
+      },
+      help =
+          "Whether to narrow inputs to C/C++ compilation by scrapping dependencies from input"
```

Comment:
nit: is this supposed to be "scraping dependencies"? Might be worth rephrasing either way.

Reply:
Good catch. I will replace it with "parsing".